patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -80,15 +96,23 @@
              }
              else
              {
-@@ -1707,6 +1720,7 @@
+@@ -1382,6 +1397,7 @@
+             this.field_70126_B -= 360.0F;
+         }
+ 
++        this.field_70170_p.func_72964_e((int) Math.floor(this.field_70165_t) >> 4, (int) Math.floor(this.field_70161_v) >> 4); // Forge - ensure target chunk is loaded.
```

Comment:
The getChunk line was specifically placed here to reduce it to only player controlled movement changes, and not the setPosition method that is called for literally everything (which would hurt performance)

This line is very important to happen BEFORE the .setPosition called right below it. the setPosition following this will then call the updateEntityWithOptionalForce method.

Both are occurring, but the behavior of updateEntityWithOptionalForce changes if the target chunk is not loaded, so we force load it here if it was not (teleporting)